piccolo/apps/user/tables.py

```diff
@@ -50,9 +50,24 @@ def __init__(self, **kwargs):
         # Generating passwords upfront is expensive, so might need reworking.
         password = kwargs.get("password", None)
         if password:
+            if password.startswith("pbkdf2_sha256") and len(password) == 117:
```

Comment:
I think that only first check `if password.startswith("pbkdf2_sha256")` will be enough.

Reply:
What if the user intentionally enters a password that starts with "pbkdf2_sha256"? The `len(password)` is to differentiate between if a value was hashed(`hash_password` returns a password of length 117) or just a value entered by the user that starts with "pbkdf2_sha256"?

Should I still remove the condition?

Reply:
I think both have valid arguments. On the one hand checking the length provides extra confidence. But we can't guarantee that the length will always be 117 (we might increase the default salt size and / or number of iterations in the future). So I think it's OK to remove it.

Reply:
Okay.